Simplified/Simplified.js

```diff
@@ -152,6 +158,10 @@ function Simplified() {
   this.linkOpenDyslexicFonts = function() {
     var id = 'simplified-opendyslexic';
     var innerDocument = window.frames['epubContentIframe'].document;
+    if (!innerDocument) {
```

Comment:
Should we also check the OS along with a lack of `innerDocument`? Is there any possibility of other changes to the app down the road causing this check to actually be ambiguous with the assumption that it's guaranteeing an iOS level?

Reply:
I'm going to leave this as-is, as discussed!